MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.RadioButton.xaml

```diff
@@ -595,7 +595,7 @@
               </VisualStateGroup>
             </VisualStateManager.VisualStateGroups>
             <Border Grid.ColumnSpan="2"
-                    Background="{DynamicResource MaterialDesignChipBackground}"
+                    Background="{DynamicResource MaterialDesign.Brush.Chip.Background}"
```

Comment:
Should the be target type specific?

In line 789 below you have a `RadioButton` specific `Chip` brush.

```suggestion
                    Background="{DynamicResource MaterialDesign.Brush.RadioButton.Chip.Background}"
```

Reply:
In this case I don't think so because the radio button is being intentionally styled to look like the chip so I think the chip brush makes more sense.